code/game/objects/structures/ai_core.dm

```diff
@@ -144,7 +144,7 @@
 					user << "<span class='warning'>This MMI does not seem to fit!</span>"
 					return
 
-				if(M.hacked)
+				if(M.hacked || M.clockwork)

```

Comment:
I mean, I could, but I don't want to fuck with this code right now so hey, stopgap fixes